notebooks/json_blocks/WET/BLOCK-T414.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>I think all the same comments as the notebook above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>